### ISSUE TITLE
Error when using multiple Gpus

### ISSUE THREAD

**Issue by xihuai18** [lonePatient/chinese-word2vec-pytorch#1]
![image](https://user-images.githubusercontent.com/23721828/56081249-c1dccd80-5e3d-11e9-976b-a4a16209472c.png)

**Comment by lonePatient** [comment]
@Leo-xh hi,for multi gpus, you try :
if len(n_gpus)>1:
    loss = loss.means()
loss.backward()
